### PR TITLE
Improve create profile UI

### LINKED ISSUE #62: Add a better UI for the profile creation: Change the font and colors + Handle exceptions
- [x] Changed font
- [x] Changed Color
- [x] Updated the drop down menu
- [x] Handled exceptions
- [x] Tests

### PR DESCRIPTION
**Pull request by yasmineben510**
Changed the UI of the profile: 
- Changed the font and the colors,
- Added logos of the service providers to choose from
- Added the exception handling : if an exception is raised, the UI displays an informative message to the user
- Modified the button to display "Creating" while waiting for the server's answer
- Added tests for these error handling functionalities and get 80% coverage

Below is a preview of the new UI
Capture d’écran 2023-03-25 à 14 40 12

Capture d’écran 2023-03-25 à 14 40 05

This closes #62

### CODE REVIEW

File: app/src/androidTest/java/com/github/orkest/View/auth/SignUpFormTest.kt [hunk 1]
```diff
@@ -130,4 +130,26 @@ class SignUpFormTest {
...
+
+    @Test
+    fun emptyUsernameDisplaysError(){
+        composeTestRule.onNodeWithText("Username")
+            .performTextInput("")
+
+        composeTestRule.onNodeWithText("Create Profile").performClick()
+
+        composeTestRule.onNodeWithText("Username cannot be empty!").assertIsDisplayed()
+    }
```
Comment (evguenirousselot) [inline app/src/androidTest/java/com/github/orkest/View/auth/SignUpFormTest.kt:142]: Nice that you added empty error handling, it was missing in the previous sprint

File: app/src/main/java/com/github/orkest/ViewModel/auth/AuthViewModel.kt [hunk 2]
```diff
@@ -90,26 +86,21 @@ open class AuthViewModel: ViewModel() {
...
-            //Propagates the exception in case of another exception
+            } //Propagates the exception in case of another exception
             .addOnFailureListener{
                 future.completeExceptionally(
-                    Exception("Sorry, something went wrong ... Please check your Internet connection"))
+                Exception("Sorry, something went wrong ... Please check your Internet connection"))
             }
-
         return future
     }
```
Comment (evguenirousselot) [inline app/src/main/java/com/github/orkest/ViewModel/auth/AuthViewModel.kt:105]: I thought you've reduced the number of lines by improving the function, maybe be a bit more explicit in the commit message

**Comment by evguenirousselot** [comment]
Very good coding and very nice new design. Maybe we could merge your part and then I can work again on mine since you've changed the design and a few function implementations.

File: app/src/main/java/com/github/orkest/ViewModel/auth/AuthViewModel.kt (hunk 2 above)
Comment (yasmineben510) [reply app/src/main/java/com/github/orkest/ViewModel/auth/AuthViewModel.kt:105]: Okay will do next time thanks!

**Review by evguenirousselot — APPROVED** [review]

**Pull request merged**